redash/models/__init__.py

```diff
@@ -67,6 +67,56 @@ def get(self, query_id):
 scheduled_queries_executions = ScheduledQueriesExecutions()
 
 
+@python_2_unicode_compatible
+class TableMetadata(db.Model):
```

Comment:
you'll need a migration to create these tables

Reply:
Ah! I had missed this, thank you!